internal/cluster/validator.go

```diff
@@ -465,3 +466,25 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) isOCSMinRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
```

Comment:
Here as I mentioned have a generic validation method for all operators, then OCS-specific stuff only in internal/ocs